README.md

```diff
@@ -843,13 +843,45 @@ Params:
 - `steps`: a list of steps/actions to execute in the scope of the for_each loop.
   It's analogous to the `steps` field at the top level of the spec file.
 
+### Ignore (Optional)
+
+We use [filepath Glob](https://pkg.go.dev/path/filepath#Glob) to match the file
+and directory paths that should be ignored if included/copied from template
+directory to destination directory, and should not be overwritten if included
```

Comment:
Please correct me if it is not the desired behavior, an alternative is to remove the ignored destination files, but that seems risky.

Reply:
Make sense, updating.